pyproject.toml

```diff
@@ -17,10 +17,12 @@ flake8 = "^4.0"
 coverage = "^6.3"
 isort = "^5.6.3"
 black = "^21.7b0"
-mypy = "^0.942"
+mypy = "^0.950"
 pytest = "^7.0.0"
 pytest-sugar = "^0.9.4"
 pytest-cov = "^3.0.0"
+click = "8.0.4"
```

Comment:
nit: probably update branch of [multiple-testing-calculator](https://github.com/godaddy/sample-size/tree/multiple-testing-calculator), so we know it is not an update you made in this PR?